src/themes/store/styles/textStyles.js

```diff
@@ -39,9 +39,8 @@ export const textStyles = {
   },
   lightCaption: {
     ...headingsBase,
-    fontSize: 'xs',
-    lineHeight: '18px',
+    fontSize: 'sm',
+    lineHeight: '17px',
```

Comment:
I noticed the rest of them are all `18px` but the designers told me to use `17px` so not sure if we wanna change this to `18px` for consistency or change the other ones 🤔

Reply:
see comments here: https://www.notion.so/uwblueprintexecs/Storefront-Shopping-Bag-fe1a95f06de54e36a864d4a6e4ea11c1

Reply:
probably a typo, I would follow figma https://www.figma.com/file/LF4zDBXci16J48b9ki6zw6/Building-Up---Storefront?node-id=189%3A3296, no that this is really noticeable at all